src/dra/5D6C4.c

```diff
@@ -318,8 +318,8 @@ void GetEquipProperties(s32 handId, Equipment* res, s32 equipId) {
 
     res->criticalRate = criticalRate;
     func_800F4994();
-    damageScale = D_800A4B04[equipId].damageScale;
-    if (damageScale != 6 && damageScale != 10) {
+    itemCategory = D_800A4B04[equipId].itemCategory;
+    if (itemCategory != 6 && itemCategory != 10) {
```

Comment:
Use the enum here? There's probably other spots in the codebase as well.

Reply:
Thanks, missed that one. Pretty sure that's all of them for now.